internal/transformer/transformer.go

```diff
@@ -80,6 +80,9 @@ func init() {
 		new(generators.ReadMeGenerator),
 
 		new(dockerfilegenerators.NodejsDockerfileGenerator),
+		new(dockerfilegenerators.WinConsoleAppDockerfileGenerator),
```

Comment:
Would be good to put all dotnet ones in a new package below dockerfilegenerators to keep them together.

Reply:
done